lib/features/navigation/service/router.dart

```diff
@@ -13,7 +16,7 @@ part 'router.gr.dart';
 /// Main point of the application navigation.
 /// {@endtemplate}
 @AutoRouterConfig(
-  replaceInRouteName: 'Flow,Route',
+  replaceInRouteName: 'Flow|Screen,Route',
```

Comment:
You should leave just `Flow` here and add DebugFlow and UiKitFlow as I mentioned above.